bitnami/phpmyadmin/README.md

```diff
@@ -78,6 +78,8 @@ The following table lists the configurable parameters of the phpMyAdmin chart an
 | `ingress.hosts[0].tls`       | Utilize TLS backend in ingress                                                                          | `false`                                                      |
 | `ingress.hosts[0].tlsHosts`  | Array of TLS hosts for ingress record (defaults to `ingress.hosts[0].name` if `nil`)                    | `nil`                                                        |
 | `ingress.hosts[0].tlsSecret` | TLS Secret (certificates)                                                                               | `phpmyadmin.local-tls-secret`                                |
+| `extraEnv`                   | Any extra environment variables to be passed to the pod (evaluated as a template)                       | `{}`                                                         |
+| `extraEnvVarsCM`             | Name of a Config Map containing extra environment variables to be passed to the pod (evaluated as a template) | `nil`                                                  |
```

Comment:
Also add extraEnvVarsSecret for the cases that we want to put sensitive information.